Cork/Views/Sidebar/Components/Casks Section.swift

```diff
@@ -9,100 +9,38 @@ import SwiftUI
 
 struct CasksSection: View {
     
-    @AppStorage("allowMoreCompleteUninstallations") var allowMoreCompleteUninstallations: Bool = false
-    
     @EnvironmentObject var appState: AppState
     @EnvironmentObject var brewData: BrewDataStorage
-    @EnvironmentObject var outdatedPackageTracker: OutdatedPackageTracker
     
-    @Binding var searchText: String
+    let searchText: String
     
     var body: some View {
         Section("sidebar.section.installed-casks")
         {
-            if !appState.isLoadingCasks
+            if appState.isLoadingCasks
```

Comment:
Flipped the conditional branches in this view, and the formulae section – it's a little easier to understand without a negation at first glance